opm/simulators/wells/ALQState.hpp

```diff
@@ -0,0 +1,59 @@
+/*
+  Copyright 2021 Equinor ASA
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef OPM_ALQ_STATE_HEADER_INCLUDED
+#define OPM_ALQ_STATE_HEADER_INCLUDED
+
+#include <map>
+#include <string>
+#include <vector>
+
+
+namespace Opm {
+
+class ALQState {
+public:
+
+    std::size_t pack_size() const;
```

Comment:
Please include `<cstddef>` to get a declaration for `std::size_t`.

Reply:
ok